docs/antrea-network-policy.md

```diff
@@ -254,6 +254,40 @@ spec:
           port: 5978
       name: DropToThirdParty
       enableLogging: true
+---
+apiVersion: crd.antrea.io/v1alpha1
```

Comment:
maybe we should have named sub-sections for these examples now, for easier readability and navigation

Reply:
are you referring to splitting these yaml specs in to sub sections or a bit more involved change with sub sections for fields and set the example yaml within that sub section i.e. a new sub section for "Namespaces" field and then place this yaml there? I am hoping for the former :)